### ISSUE TITLE
Not available in VS2019

### ISSUE THREAD

**Issue by improwise** [didourebai/BlogTemplateAngular#1]
Despite being installed in VS2019 (updated with .net core 2.2 installed) and verified as enabled under extentions, it can't be found when trying to create a new project in VS2019 (latest version).

**Comment by abukres** [comment]
Same here. The template is now showing.

**Comment by didourebai** [comment]
Are you able to display the project in VS2019 ? You still have a problem?

**Comment by abukres** [comment]
It does not show up. I used two computers. Make sure you're using the latest VS 2019 update.
Your instructions say to make sure npm is 8.x. The latest npm version in 6.11.3.

**Comment by didourebai** [comment]
@abukres yes, you are right npm 6.x, not 8.x, I will update this but it is strange that you are not able to find the template else try to download t from the marketplace: https://marketplace.visualstudio.com/items?itemName=RebaiHamidaTrabelsi.Angular8.
![3](https://user-images.githubusercontent.com/11868839/65206522-a1541780-da5c-11e9-9b86-576924ce867e.JPG)
![2](https://user-images.githubusercontent.com/11868839/65206523-a1541780-da5c-11e9-917b-d79c57ac7a68.JPG)
![1](https://user-images.githubusercontent.com/11868839/65206524-a1541780-da5c-11e9-9886-1a64dfcf0c94.JPG)

**Comment by abukres** [comment]
It shows up under Manager Extensions but not when creating a new project. It's not one of the project templates when I search for 'Angular'.
